spec/models/manageiq/providers/kubernetes/container_manager/container_node_spec.rb

```diff
@@ -32,4 +32,37 @@
     )
     expect(node.container_images.count).to eq(1)
   end
+
+  describe "#external_logging_path" do
+    def get_query(path)
+      index = ".operations.*"
+      prefix_len = "/#/discover?_g=()&_a=(columns:!(hostname,level,kubernetes.pod_name,message),filters:!((meta:(disabled:!f,index:'#{index}',key:hostname,negate:!f),".length
```

Comment:
It feels wrong to duplicate this string in the test.  Is there a way to keep it in the model?  Perhaps a pair of methods for constructing and desconstructing the query string in the model?